jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/JsonbTransientDiagnostic.java

```diff
@@ -31,13 +31,14 @@
     private String favoriteLanguage;    // Diagnostic: JsonbTransient is mutually exclusive with other JsonB annotations
     
     @JsonbProperty("person-id")
+    @JsonbNillable
     private int getId() { 
         // A diagnostic is expected on getId because as a getter, it is annotated with other 
         // Jsonb annotations while its corresponding field id is annotated with JsonbTransient
         return id;
     }
     
-    @JsonbNillable 
+    @JsonbNillable
```

Comment:
Add the corresponding import statement to this class

Reply:
Looks like this class also needs the `@JsonbAnnotaiton`  import statement

Reply:
Thanks for catching that, I added the import statements and modified the line numbers in the test.